wayfindingsample/src/main/java/mappedin/com/wayfindingsample/MainActivity.java

```diff
@@ -491,36 +632,6 @@ public void onClick(DialogInterface dialog, int which) {
 
         showWelcomePage();
 
-        final MappedinCallback<Venue> getVenueCallback = new MappedinCallback<Venue>() {
-            @Override
-            public void onCompleted(Venue venue) {
-                if (venue != null){
-                    addStoreLabel(venue);
-                    enableSearch(venue);
-                    maps = venue.getMaps();
-                    Arrays.sort(maps, new Comparator<Map>() {
-                        @Override
-                        public int compare(Map left, Map right) {
-                            return right.getFloor() - left.getFloor();
-                        }
-                    });
-                    for (int i = 0; i < maps.length; i ++) {
-                        levelFloorMap.put(maps[i].getFloor(), i);
-                    }
-                    mapListAdapter =
-                            new MapListAdapter(MainActivity.this, context, maps);
-                    levelPickerListView.setAdapter(mapListAdapter);
-                    setMap(maps.length-1);
-                    iAmHereFloorIndex = maps.length-1;
-                }
-            }
-
-            @Override
-            public void onError(Exception exception) {
-
-            }
-        };
-
         MappedinCallback<List<Venue>> getVenuesCallback = new VenuesCallback(mappedIn, getVenueCallback);
```

Comment:
maybe we should make this getVenuesCallback reusable so we can use it in your onItemClicked